report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,60 @@
 \section{Problem Definition}\label{sec:problem_definition}
+The primary objective of this research is to enhance computational methods for the accurate and robust quantification of chemical compositions using \gls{libs} data.
+This enhancement seeks to improve the adaptability and precision of \gls{libs} across various environmental contexts, including terrestrial and planetary geological samples.
 
```

Comment:
We've mentioned multicollinearity and matrix effects in the introduction. This sentence reads a little like its the first time we've presented it. Does it perhaps make sense to rephrase it so that its clear that its not the first time we mention it?